skl2onnx/_supported_operators.py

```diff
@@ -193,11 +193,6 @@
 # each cluster in most cases.
 cluster_list = [KMeans, MiniBatchKMeans]
 
-# Classifiers with converters supporting decision_function().
```

Comment:
I don't think we should remove this unless we support all the classifiers with decision_function(). I am working on it though.

Reply:
I think we should test whether or not an option is available but not add a specific mechanism for classifiers supporting decision_function. That's the first Reason I removed it, the second reason is we forgot to update it. So it was wrong. I really prefer to address the fact we check option. It is more generic.

Reply:
CalibratedClassifierCV doesn't support decision_function() itself, so it's not about checking options that user passes. It should always call decision_function() if the base_classifier has decision_function(), else it should call predict_proba(). I added this check since, we didn't support decision_function for most. Now that we have that, we should add a check if the base_classifier has decision_function attribute and if it does call them accordingly.

Reply:
Since, you are doing this:
container.add_options(id(base_model), {'raw_scores': True})
we don't need to check for the presence of decision_function(), as we only use this(check raw_scores) in classifiers converters that have decision_function. We may want to note that here though(in comments).

Reply:
See PR #380.